src/view/renderer.js

```diff
@@ -103,6 +103,13 @@ export default class Renderer {
 		 */
 		this.isFocused = false;
 
+		/**
+		 * Indicates if composition takes places inside view document.
+		 *
+		 * @member {Boolean}
+		 */
+		this.isComposing = false;
```

Comment:
Why this is needed here? It is not used anywhere in the renderer.

Reply:
Not needed, it was a leftover (well, it probably will be used in the related task, but definitely should not be added in this PR) - removed.